database/src/factories/login-user-roles.factory.ts

```diff
@@ -0,0 +1,16 @@
+import Faker from 'faker'
+import { define } from 'typeorm-seeding'
+import { LoginUserRoles } from '../../entity/LoginUserRoles'
+import { LoginUserRolesContext } from '../interface/UserContext'
+
+define(LoginUserRoles, (faker: typeof Faker, context?: LoginUserRolesContext) => {
+  if (!context) context = {}
+  if (!context.userId) throw new Error('LoginUserRoles: No userId present!')
+  if (!context.roleId) throw new Error('LoginUserRoles: No roleId present!')
```

Comment:
Same here, i suggest to just put this all together into one validation block. This might reduce the granularity of the errors tho.